docs/source/contributing/documentation.rst

```diff
@@ -317,3 +317,75 @@ elements that are either a ``str`` or ``None``;
 ``Tuple[:class:`str`, :class:`int`]`` for a tuple of type
 ``(str, int)``; ``Tuple[:class:`int`, ...]`` for a tuple of variable
 length with only integers.
+
+
+Adding type hints to functions and parameters
+---------------------------------------------
+
+If you've never used type hints before, this is a good place to get started:
+https://realpython.com/python-type-checking/#hello-types.
+
+When adding type hints to manim, there are some guidelines that should be followed:
+
+* Coordinates have the typehint ``Sequence[float]``, e.g.
```

Comment:
We can also define some manim specific type aliases... so that there could be a Type Hint `Coordinates`.
Furthermore, it would give us a central location to narrow the type in future.

Reply:
Yess, sounds like a really good idea!
Should they go then into the `constants.py` folder?

Reply:
we should probably have a separate typing.py if we're going down this route.